core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7568SettingsProfileDeactivationTest.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.maven.it;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.it.util.ResourceExtractor;
+import static org.apache.maven.it.ItUtils.verifyTextNotInLog;
+
+import java.io.File;
+
+public class MavenITmng7568SettingsProfileDeactivationTest
+        extends AbstractMavenIntegrationTestCase
+{
+    private static final String PROJECT_PATH = "/mng-7568-inactive-profile-deactivation";
+
+    public MavenITmng7568SettingsProfileDeactivationTest()
+    {
+        super( "[3.0.4,4)" );
```

Comment:
This does make sense if it is in 3.8.x only.

Reply:
https://github.com/apache/maven-integration-testing/pull/205#discussion_r997920967